CHANGELOG.md

```diff
@@ -23,6 +23,33 @@ This change was made in [pull request #3599: Enable new link styles by default](
 
 ### Breaking changes
 
+#### "Compatibility mode" features are no longer supported
+
+GOV.UK Frontend no longer supports compatibility with legacy frameworks. You can no longer incrementally add GOV.UK Frontend to an existing service using a previous GOV.UK framework. We have removed the following Sass variables and one mixin which controlled compatibility mode:
```

Comment:
What's a legacy framework? 🤷‍♂️

It's been so long since we told people to migrate, should we spell it out and link here?
https://frontend.design-system.service.gov.uk/migrating-from-legacy-products/

```suggestion
GOV.UK Frontend no longer supports compatibility with our old frameworks, GOV.UK Elements, GOV.UK Template and GOV.UK Frontend Toolkit. You can no longer incrementally add GOV.UK Frontend to an existing service using a previous GOV.UK framework. We have removed the following Sass variables and one mixin which controlled compatibility mode:
```

Reply:
We fondly call them "our old frameworks" rather than _legacy_

Reply:
I was so sure we called them "legacy frameworks" 🤔 I'll amend. I think it's a good idea to link to that page but I'm not sure how the docs split work is going to impact the freshness of that URL. We should be cognisant of this and maybe return to this changelog item later.